content/en/docs/Testing Policies/_index.md

```diff
@@ -3,9 +3,89 @@ title: "Testing Policies"
+
+The second use case is when ensuring a set of policies and resources always produce a predefined set of outcomes. This is useful when testing strategic modifications to either policies and/or resources, or when testing future versions of Kyverno prior to upgrading a cluster. For example, operations teams may want to ensure that a standard set of platform services are always allowed to be created in a cluster by defining a Kyverno test manifest which declares they should pass. Those services, represented in YAML manifest form, may be strategically changed when, for example, a new image version is available. By executing the same tests, expected results may be compared to actual results. If there are discrepancies, the tests fail and operations teams will know which test failed and can resolve the issue prior to changes made in a cluster.
+
+### GitHub Actions
+
+[GitHub Actions](https://github.com/features/actions) provides a comprehensive CI system for repositories hosted on GitHub. Tests for Kyverno policies may be executed via a workflow as part of the CI process.
+
+The following is an example of a simple GitHub Actions workflow which may be used as part of building a larger CI pipeline. Consider a repository with the following structure.
+
+```
+.
+├── .github
+│   └── workflows
+│       └── kyverno.yaml
+├── README.md
+├── policies
+│   ├── disallow-privilege-escalation.yaml
+│   └── disallow-privileged-containers.yaml
+├── resources
+│   ├── configmap.yaml
+│   └── deployment.yaml
+└── tests
+    ├── require_labels
+    │   ├── kyverno-test.yaml
+    │   ├── require_labels.yaml
+    │   └── resource.yaml
+    └── restrict_image_registries
+        ├── kyverno-test.yaml
+        ├── resource.yaml
+        └── restrict_image_registries.yaml
+```
+
+The repository contains Kyverno policies stored in `policies/`, Kubernetes resource manifests stored in `resources/` and complete Kyverno test cases stored in `tests/`. A sample workflow may be added at the path `.github/workflows/kyverno.yaml` with the below contents which performs the following for both opened pull requests and manually if triggered on the repository.
+
+1. The repository is checked out.
+2. The Kyverno CLI is downloaded by naming a specific version. This can be changed to simulate the effect a Kyverno upgrade may have in a cluster.
+3. Tests the unknown manifests in `resources/` against the known policies stored in `policies/`.
+4. Tests the pre-defined test cases stored in `tests/` which contain Kyverno test manifests, policies, and resources separated by folder.
+
+```yaml
+name: kyverno-policy-test
+on:
+  - pull_request
+  - workflow_dispatch
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Download Kyverno CLI
+        run: |
+          curl -sLO https://github.com/kyverno/kyverno/releases/download/v1.7.2/kyverno-cli_v1.7.2_linux_x86_64.tar.gz
```

Comment:
```suggestion
          export kyvernoversion = v1.7.2
          curl -sLO https://github.com/kyverno/kyverno/releases/download/${kyvernoversion}/kyverno-cli_${kyvernoversion}_linux_x86_64.tar.gz
```
Not entirely sure about the formatting on top of my head. The variable can then also be used in the line below. It makes it a bit easier to keep kyverno versions up to date.

Reply:
I'd rather move this to an env var at the top of the action which is easily done.